src/components/views/rooms/RoomTile.tsx

```diff
@@ -451,6 +462,16 @@ export default class RoomTile extends React.PureComponent<IProps, IState> {
             const isLowPriority = roomTags.includes(DefaultTagID.LowPriority);
             const lowPriorityLabel = _t("Low Priority");
 
+            const inRoom = this.props.room && this.props.room.getMyMembership() === "join";
+            const userId = MatrixClientPeg.get().getUserId();
+            let canInvite = inRoom;
+            const powerLevels = this.props.room.currentState
```

Comment:
seems a bit strange that you're checking for the truthiness of `this.props.room` above but not here.

the `IProps` say it should always be set

Reply:
Yes, I've changed that.